### ISSUE TITLE
Version 1.3.6 is Tagged But not in available via pip

### ISSUE THREAD

**Issue by jamesward** [mehcode/python-xmlsec#89]

**Comment by bgaifullin** [comment]
there is problem with publishing builds on pip via travis. 
If anybody helps to resolve this - it will be great.

**Comment by adelton** [comment]
When installing via `pip3` on my Fedora, I get

```
Processing /root/python-xmlsec
Requirement already satisfied: pkgconfig in /usr/lib/python3.7/site-packages (from xmlsec==1.0.1.dev0) (1.3.1)
Requirement already satisfied: lxml>=3.0 in /usr/lib64/python3.7/site-packages (from xmlsec==1.0.1.dev0) (4.2.5)
Installing collected packages: xmlsec
  Running setup.py install for xmlsec ... done
Successfully installed xmlsec-1.0.1.dev0
```

The following

```
diff --git a/version.txt b/version.txt
index e752649..95b25ae 100644
--- a/version.txt
+++ b/version.txt
@@ -1 +1 @@
-1.0.1.dev0
+1.3.6
```

changes it to
```
Successfully installed xmlsec-1.3.6
```

Not sure if it's related to the Travis CI issues (any link?) but it seems like something that might be needed anyway. Would you like separate issue/PR?

**Comment by adelton** [comment]
And it also fixes the versioning for

```
python3 setup.py bdist_rpm
```

**Comment by joekohlsdorf** [comment]
Travis logs the following:
```
HTTPError: 401 Client Error: This password appears in a breach or has been compromised and cannot be used. See the FAQ entry at https://pypi.org/help/#compromised-password for more information. for url: https://upload.pypi.org/legacy/
```

@mehcode Please reset the password of the project in PyPi and change it accordingly in .travis.yml.

**Comment by mehcode** [comment]
Hopefully that fixes it. Re-open if there is still an issue.

**Comment by rouge8** [comment]
The latest release available on PyPI is still 1.3.3: https://pypi.org/project/xmlsec/#history Maybe it's time for a 1.3.7 to get a new upload?

**Comment by hoefling** [comment]
The latest release is now 1.3.8 which resolves this issue.
